moveToWebcompat.js

```diff
@@ -0,0 +1,117 @@
+const REQUIRED = {
+  op_sys: "OS (Categories > Platform > OS)",
+  platform: "Platform (Categories > Platform)",
+  url: "URL (References > URL)",
+  version: "Version of browser (Categories > Version)",
+};
+
+const getBugId = () => {
+  const urlParams = new URLSearchParams(window.location.search);
+  return urlParams.get("id");
+};
+
+const openDropDown = (text, elms) => {
+  const dropdown = document.querySelector("#bw-move .dropdown-content");
+
+  if (!dropdown) return;
+  if (text) dropdown.innerText = text;
+  if (elms && elms.length) elms.forEach((el) => dropdown.appendChild(el));
+
+  dropdown.style.display = "block";
+
+  const hideDropDown = function (event) {
+    const withinBoundaries = event.composedPath().includes(dropdown);
+
+    if (!withinBoundaries) {
+      dropdown.style.display = "none";
+      document.removeEventListener("click", hideDropDown);
+    }
+  };
+
+  setTimeout(() => document.addEventListener("click", hideDropDown), 100);
+};
+
+const getBugData = (bugId) => {
+  return fetch(
+    `https://bugzilla.mozilla.org/rest/bug/${bugId}?include_fields=url,platform,op_sys,version`
+  )
+    .then((response) => response.json())
+    .then((data) => {
+      return data;
+    })
+    .catch((error) => {
+      openDropDown(error);
+    });
+};
+
+const getRequired = (bugData) => {
+  const required = [];
+
+  for (const [key, value] of Object.entries(bugData)) {
+    if (key in REQUIRED && (!value || value.toLowerCase() === "unspecified")) {
+      const errList = document.createElement("li");
+      errList.innerText = REQUIRED[key];
+      errList.classList.add("bw-error");
+      required.push(errList);
+    }
+  }
+
+  return required;
+};
+
+const onMoveButtonClick = async () => {
+  const bugId = getBugId();
+  if (!bugId)
+    openDropDown("A valid bug id can't be found. Please check the url.");
```

Comment:
```suggestion
    openDropDown("A valid bug ID could not be found. Please check you're actually viewing a bug.");
```

Might be easier to understand if things go wrong.